frontend/src/app/shared/models/user/userUpdate.ts

```diff
@@ -0,0 +1,6 @@
+export interface UserUpdateDto {
```

Comment:
Why do we have two of the same interfaces?

Reply:
I was thinking to use it to change the profile. Delete UserUpdateDto)